src/Lightning/Node/Api.hs

```diff
@@ -32,6 +32,11 @@ data ApiV1 route = ApiV1
       :> "genInvoice"
       :> ReqBody '[JSON] InvoiceReq
       :> Post '[JSON] InvoiceRep
+  , _listInvoices ::route
+      :- "invoice"
+      :> "listInvoices"
+      :> ReqBody '[JSON] InvoiceLabel
```

Comment:
According to [the docs](https://github.com/Ride-The-Lightning/c-lightning-REST/blob/566a41466fdbd03bd96450bfc8742eb64bedb753/controllers/invoice.js#L104), the label is passed not in the request body, but as a GET argument (i.e. `QueryParam`). Also note that a `QueryParam` will automatically get wrapped in a `Maybe`.

Reply:
Ok^ you answered to my previous comment